src/components/component-playground.jsx

```diff
@@ -180,15 +179,41 @@ module.exports = React.createClass({
     </ul>;
   },
 
+  _renderPreview: function() {
+    return <div ref="previewContainer" className={this._getPreviewClasses()}>
+      {this.loadChild('preview')}
+    </div>
+  },
+
   _renderContentFrame: function() {
     return <div ref="contentFrame" className={this._getContentFrameClasses()}>
-      <div ref="previewContainer" className={this._getPreviewClasses()}>
-        {this.loadChild('preview')}
-      </div>
-      {this.props.editor ? this._renderFixtureEditor() : null}
+      {this.props.editor ? this._renderSplitPane() : this._renderPreview()}
     </div>
   },
 
+  _renderSplitPane: function() {
+    return <SplitPane
+      split={this._getOrientationDirection()}
+      ref='editorPreviewSplitPane'
+      defaultSize={localStorage.getItem('splitPos')}
+      onChange={size => localStorage.setItem('splitPos', size)}
+      minSize={20}
+      className={this._getSplitPaneClasses('split-pane')}
+      resizerClassName={this._getSplitPaneClasses('resizer')}
+    >
+      {this._renderFixtureEditor()}
+      {this._renderPreview()}
+    </SplitPane>
+  },
+
+  _getOrientationDirection: function() {
+    return this.state.orientation == 'landscape' ? 'vertical' : 'horizontal';
```

Comment:
You can do in three different ways:
1. Make `react-split-pane` to recognize the `orientation` for `landscape` and `portrait` meaning that you should change the `vertical, horizontal` split to `landscape, portrait` mode.
2. Change playground orientation from `landscape, portrait` to `vertical, horizontal`
~~3. leave it like this.~~